platform/platform-scheduler/src/main/java/ddf/platform/scheduler/ScheduledTask.java

```diff
@@ -28,21 +28,21 @@
     /**
      * Creates and schedules new task
      */
-    public void newTask();
+    void newTask();
 
     /**
      * Removes a task completely so that it does not run or exist.
      *
      * @param code - not used
      * @see https://issues.apache.org/jira/browse/ARIES-1436
      */
-    public void deleteTask(int code);
+    void deleteTask(int code);
 
     /**
      * Updates an existing task with new properties.
      *
      * @param properties
      */
-    public void updateTask(Map properties);
+    void updateTask(Map<String, Object> properties);
```

Comment:
It's worth noting that this is used by blueprint for the `update-method`.  We make the map types explicit in other update-method calls.  https://github.com/codice/ddf/blob/master/catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java#L604